### DISCUSSION TITLE
Unable to install Ruby 2.7.7 using rbenv on Apple Silicon M1 Ventura 13.2.1

### DISCUSSION THREAD

**Discussion by ankgautam** [rbenv/ruby-build#2162] [Build failures]
When i Ran this command :
```
CONFIGURE_OPTS=--with-openssl-dir=`brew --prefix openssl@1.1` CFLAGS="-Wno-error=implicit-function-declaration" rbenv install 2.7.7
```
Error:
```
To follow progress, use 'tail -f /var/folders/69/5cg3_lw51p73dyb7m4q9bxw80000gn/T/ruby-build.20230309171330.30923.log' or pass --verbose
Downloading ruby-2.7.7.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.7.tar.bz2
Installing ruby-2.7.7...

WARNING: ruby-2.7.7 is nearing its end of life.
It only receives critical security updates, no bug fixes.

ruby-build: using readline from homebrew
ruby-build: using libyaml from homebrew
ruby-build: using gmp from homebrew

BUILD FAILED (macOS 13.2.1 using ruby-build 20230309)

Inspect or clean up the working tree at /var/folders/69/5cg3_lw51p73dyb7m4q9bxw80000gn/T/ruby-build.20230309171330.30923.hLgbRM
Results logged to /var/folders/69/5cg3_lw51p73dyb7m4q9bxw80000gn/T/ruby-build.20230309171330.30923.log

Last 10 log lines:

********************
make[2]: *** [cparse.o] Error 254
make[1]: *** [ext/racc/cparse/all] Error 2
make[1]: *** [ext/json/all] Error 2
linking shared-object fiddle.bundle
linking shared-object -test-/string.bundle
linking shared-object fiber.bundle
make[1]: *** [ext/psych/all] Error 2
make: *** [build-ext] Error 2
```

**Comment by mislav** [comment]
Hi, if you look at the `Results logged to` line, it points to a log file. In that log file you can find the full build log. See if you can isolate the first line that says something like "error" or "fail".

The last 10 lines of log that you pasted do not contain the information about what caused the actual build failure.

**Reply by jiayongchang** [reply]
install ./apps/tsget.pl -> /Users/mx/.rbenv/versions/2.7.6/openssl/ssl/misc/tsget.pl
link /Users/mx/.rbenv/versions/2.7.6/openssl/ssl/misc/tsget -> /Users/mx/.rbenv/versions/2.7.6/openssl/ssl/misc/tsget.pl
install ./apps/openssl.cnf -> /Users/mx/.rbenv/versions/2.7.6/openssl/ssl/openssl.cnf.dist
install ./apps/openssl.cnf -> /Users/mx/.rbenv/versions/2.7.6/openssl/ssl/openssl.cnf
install ./apps/ct_log_list.cnf -> /Users/mx/.rbenv/versions/2.7.6/openssl/ssl/ct_log_list.cnf.dist
install ./apps/ct_log_list.cnf -> /Users/mx/.rbenv/versions/2.7.6/openssl/ssl/ct_log_list.cnf
/var/folders/1j/fw290_7x16ngbzbswt4416240000gn/T/ruby-build.20230706161846.96264.MGAhix ~/.rbenv/plugins
~/.rbenv/plugins
/var/folders/1j/fw290_7x16ngbzbswt4416240000gn/T/ruby-build.20230706161846.96264.MGAhix ~/.rbenv/plugins
curl: (56) Recv failure: Connection reset by peer

**Reply by mislav** [reply]
@jiayongchang This looks like a network issue: `curl: (56) Recv failure: Connection reset by peer`
